specification/compute/resource-manager/readme.md

```diff
@@ -829,6 +829,23 @@ input-file:
 - Microsoft.ContainerService/preview/2015-11-01-preview/containerService.json
 ```
 
+### Tag: package-2021-06-01-preview
+
+These settings apply only when `--tag=2021-06-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-06-01-preview'
+input-file:
+- Microsoft.Compute/preview/2021-06-01-preview/diagnostic.json
+```
```

Comment:
We need to include other compute swaggers in this tag too.

Reply:
Added swaggers of package 2021-03-01 under tag-package-2021-06-01-preview